cinema_service/settings.py

```diff
@@ -20,12 +20,11 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = ("django-insecure-6vubhk2$++agnctay"
+              "_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3")
```

Comment:
The `SECRET_KEY` should not be hardcoded in the settings file. Use environment variables to securely manage sensitive information.